src/bin/rr

```diff
@@ -132,7 +132,7 @@ class RRHelper
      * @param string $target
      * @throws Exception
      */
-    public static function extractBinary(string $archive, string $target)
+    protected static function extractZIP(string $archive, string $target)
```

Comment:
Incorrect method.

Reply:
oops!

Reply:
if would like to make it static, can you please do the same for TAR method